code/modules/surgery/advanced/lobotomy.dm

```diff
@@ -48,7 +48,7 @@
 		span_notice("[user] begins to perform a lobotomy on [target]'s brain."),
 		span_notice("[user] begins to perform surgery on [target]'s brain."),
 	)
-	display_pain(target, "Your head pounds with unimaginable pain!", mood_event_type = /datum/mood_event/surgery)
+	display_pain(target, "Your head pounds with unimaginable pain!", mood_event_type = /datum/mood_event/surgery/success)
```

Comment:
this is backwards, this should be the failure state if anything. this also shouldn't be causing an issue, something else is wrong here causing the success/failure procs not to update the mood properly. investigating now.